src/components/InlineEdit/InlineEdit.tsx

```diff
@@ -51,17 +53,26 @@ function InlineEdit(props) {
   const inputRef = React.createRef<HTMLInputElement>();
   const textRef = useRef<HTMLSpanElement>(null);
 
-  const { onSetText } = props;
-
   useOnClickOutside(wrapperRef, () => {
     if (isInputActive) {
       save();
     }
   });
 
   function save() {
-    onSetText(inputValue);
-    setIsInputActive(false);
+    if (inputValue.length > 500) {
+      setErrorMessage("A measure name cannot be more than 500 characters.");
+    } else if (!inputValue) {
```

Comment:
Is it possible inputValue will be undefined here? Can this component be rendered without `props.text` being provided?

Reply:
A measure can never be created with an empty name. Since this is editing the measure name, I don't see a possibility to store it with a null value.

Reply:
that make more sense to swap the conditions, We might not save it, so props.text might not be null,  but the inputvalue can still be. Thanks, Joe for catching it.